cogs/utils/drawdota.py

```diff
@@ -721,6 +722,44 @@ def get_datetime_cell(match, region_data):
 		TextCell(str_time, font_size=18, horizontal_align="center")
 	)
 
+async def draw_meta_table(sorted_heroes, heroes): 
+	"""Takes a list of [heroes+p/b%+wr%] and draws a nice little discord-friendly table"""
+	
+	border_size = 10
+	grey_color = "#BBBBBB"
+	table = Table(background=discord_color2)
+	#Header
+	headers=[
+		TextCell(""), 
+		TextCell("Hero", padding=0), 
```

Comment:
I think It'd line up more with what I've been doing if you put the "Hero" header over to the left one column so that its above the images, that matches what I've got setup elsewhere in the bot.